### PR TITLE
Experiment docs

### PR DESCRIPTION
**Pull request by shwang**
Basic documentation/overview of the `experiments/` scripts.

### CODE REVIEW

File: experiments/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+### Phase 1: Download RL (PPO2) expert policies and AIRL/GAIL reward models.
+
+Use `experiments/download_models.sh`. (Downloads to `data/{expert,reward}_models/`).
+Expert policies are used in Phase 2 to generate demonstrations.
+Reward models are used in Phase 4 for transfer learning.
+
+Want to use other policies or reward models for Phase 2 or 4? 
+  * New policies can be trained using `experiments/train_experts.sh`. Enter the optional command suggested at the end of the script to upload new files to S3. Or, manually patch `data/expert_models` using the script's output files.
+  * New reward models are generated in Phase 3 `experiments/imit_benchmark.sh`. Enter the optional command suggested at the end of the script to upload new files to S3. Or, manually overwrite `data/reward_models` using the script's output files.
+Either follow the command printed by the script to update S3 or manually overwrite `data/{expert,reward}_models/`.
```
Comment (AdamGleave) [inline experiments/README.md]: Do you need this line and the explanation in the previous bullet points? Inclined to delete the last two sentences in the bullet point and expand on it in this line.

File: experiments/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+### Phase 2: Generate expert demonstrations from models.
+
+Run `experiments/rollouts_from_policies.sh`. (Rollouts saved in `data/expert_models/`).
+Demonstrations are used in Phase 3 for imitation learning.
+
+### Phase 3: Train imitation learning.
+
+Run `experiments/imit_benchmark.sh --run_name RUN_NAME`. To choose AIRL or GAIL, add the `--airl` and `--gail` flags.
```
Comment (AdamGleave) [inline experiments/README.md]: What's the default?

File: experiments/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+To analyze these results, run `python -m imitation.scripts.analyze with run_name=RUN_NAME`. Analysis can be run even while training is midway (will only show completed imitation learner's results). [Example output.](https://gist.github.com/shwang/4049cd4fb5cab72f2eeb7f3d15a7ab47)
+
+### Phase 4: Transfer learning.
+
+Run `experiments/transfer_benchmark.sh`. To choose AIRL or GAIL, add the `--airl` and `--gail` flags. Transfer rewards are loaded from `data/reward_models`.
```
Comment (AdamGleave) [inline experiments/README.md]: Default of AIRL or GAIL?

File: experiments/README.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+## Hyperparameter tuning
+
+Add a named config containing the hyperparameter search space and other settings to `src/imitation/scripts/config/parallel.py`. (`def example_cartpole_rl():` is an example).
+
+Run your hyperparameter tuning experiment using `python -m imitation.scripts.parallel with YOUR_NAMED_CONFIG inner_run_name=RUN_NAME`.
+
+Analyze imitation learning experiments using `python -m imitation.scripts.analyze with run_name=RUN_NAME source_dir=~/ray_results`.
+
+View Stable Baselines training stats on Tensorboard (available for regular RL, imitation learning, and transfer learning) using `tensorboard --log_dir ~/ray_results`. To view only a subset of Tensorboard training progress use `imitation.scripts.analyze tensorboard_filtered_launch with source_dir=~/ray_results run_name=RUN_NAME`.
```
Comment (AdamGleave) [inline experiments/README.md]: Typo: TensorBoard

File: experiments/README.md (hunk 1 above)
Comment (AdamGleave) [reply experiments/README.md]: To elaborate, I'm thinking something like:

```
  * New policies can be trained using `experiments/train_experts.sh`. 
  * New reward models are generated in Phase 3 `experiments/imit_benchmark.sh`.
+ For both scripts, you can enter the optional command suggested at the end of the script to upload new files to S3. Or, you can manually patch `data/{expert,reward}_models` using the script's output files.
```

Comment (shwang) [reply experiments/README.md]: Thanks

**Pull request merged**